### PR TITLE
Add RGBA functionality to Color Provider

### PR DESCRIPTION
**Pull request by apsylone**
I had to use this Faker data for a project. So I share this simple new functionality with the community :-)

### CODE REVIEW

File: src/Faker/Provider/Color.php [hunk 1]
```diff
@@ -91,6 +91,14 @@ public static function rgbCssColor()
     }
 
     /**
+     * @example 'rgba(0,255,122,0.8)'
+     */
+    public static function rgbaCssColor()
```
Comment (fzaninotto) [inline src/Faker/Provider/Color.php]: I'd name it `rgbaColor()`

**Comment by fzaninotto** [comment]
Great! Would you mind adding a unit test, checking that the output satisfies a regular expression?

File: src/Faker/Provider/Color.php (hunk 1 above)
Comment (apsylone) [inline src/Faker/Provider/Color.php]: I tried to respect the naming convention of the function "rgbCssColor". That's why I think this is the most appropriate. Don't you think so ?

Comment (fzaninotto) [inline src/Faker/Provider/Color.php]: right!

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**